test cases/common/58 run target/fakeburner.py

```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
+
+from __future__ import print_function
 
```

Comment:
I want all build scripts that have `python` in the shebang to be runnable both with Python 2 and Python 3. You can never be sure which one is used to run them. There are weird platforms out there and doing this removes one potential bug from happening.